airflow/gcp/operators/dataproc.py

```diff
@@ -407,7 +407,7 @@ def _build_cluster_data(self):
             cluster_data['config']['softwareConfig']['imageVersion'] = self.image_version
 
         elif self.custom_image:
-            project_id = self.custom_image_project_id if (self.custom_image_project_id) else self.project_id
+            project_id = self.custom_image_project_id if self.custom_image_project_id else self.project_id
```

Comment:
```suggestion
            project_id = self.custom_image_project_id or self.project_id
```
This is the equivalent code, but simpler. WDYT?

Reply:
Hello from Guadalajara, @mik-laj ! We're running a new contributor hackathon.

Reply:
Hello from Warsaw, Poland. I wish you lots of fun. 😸